mrp_repair_discount/__init__.py

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# © 2015 Nicola Malcontenti - Agile Business Group
```

Comment:
Replace © by Copyright, here and in all headers.

See https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L2 ase rference.